### ISSUE TITLE
Gutenberg

### ISSUE THREAD

**Issue by waybis** [billerickson/display-posts-shortcode#180]
Any plans to add Gutenberg functionality, or do you think the built-in shortcode block will be the default approach to using this plugin long term after Gutenberg is added to core?

Also do you think shortcodes still have a role to play in new wp projects or will they become redundant and only maintained to ensure backward compatibility?

**Comment by billerickson** [comment]
Yes, I do plan to add Gutenberg support, but it won't be immediate. I want to see how far the native Recent Posts block goes before deciding what direction to take this.

I think most current instances of shortcodes will be replaced with blocks in the future. Blocks are easier to use and more powerful.

**Comment by waybis** [comment]
Thanks very much for the quick reply and sharing yours plans and thoughts on Gutenberg, its good to know your plugin is going to be supported.

I like the way they have built in the shortcode & classic editor block into Gutenberg, it works really well and solves the backward compatibility issue. I expect the bigger challenge will be preventing them becoming the most popular blocks used ;)

**Comment by billerickson** [comment]
[This page on the wiki](https://github.com/billerickson/display-posts-shortcode/wiki/Gutenberg-block-ideas) is where I'm documenting my ideas for the Gutenberg block.

**Comment by waybis** [comment]
Looks great Brian, I like the way it hides next step options until they are needed.

**Comment by kauaicreative** [comment]
Looking forward to this!
